### DISCUSSION TITLE
Hello everyone!

### DISCUSSION THREAD

**Discussion by shubhe25p** [OpenMined/courses#155] [Introduce Yourself]
Hey,

This is Shubh from India, I am an undergrad researcher in ML and neuroscience. I am a beginner in AI so learning the astounding tasks it performs as well as the privacy-preserving aspect of it would be incredible. I have just watched the first lesson but the energy that Andrew carries will definitely motivate me to complete the full course. I never had the opportunity to learn from a teacher as enthusiastic as Andrew. 

Thank you to the OpenMined Team for this series, looking forward to e-meeting you

**Comment by yemikifouly** [comment]
Hello Shubh! 👋 I completely agree with what you said about Andrew! He is an extremely enthusiastic instructor and his enthusiasm is contagious. I am really glad you are enjoying the course so far!! 🙂 Welcome to OpenMined!
